src/content/docs/dropins/payment-services/containers.mdx

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
How do you get to this page? If there is no path to get to this page, let's remove it.

Reply:
I'm guessing is because of the navigation you need this page, @keharper do you know?

Reply:
Delete this file.